### PR TITLE
resolve before require

### PR DESCRIPTION
**Pull request by laggingreflex**
Fixes https://github.com/start-runner/start/issues/24

### CODE REVIEW

**Comment by deepsweet** [comment]
nice one! will test it.

File: lib/index.js [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+  if (!modulePath || !fs.existsSync(modulePath)) { // eslint-disable-line
     console.error(`Unable to find "${commander.preset}" preset, please check it again`);
     process.exit(1);
   }
 } else {
-  try {
-    tasks = require(path.resolve(commander.file));
-  } catch (e) {
+  modulePath = resolveCwd(`./${commander.file}`);
+  if (!modulePath || !fs.existsSync(modulePath)) { // eslint-disable-line
```
Comment (deepsweet) [inline lib/index.js]: trying to find a reason to have `!fs.existsSync(modulePath)` here and can't. looks like ``resolveCwd(`./${commander.file}`)`` already solves all the local path cases. part of my testing code:

```js
let modulePath = null;

if (commander.preset) {
  modulePath = resolveCwd(commander.preset);

  if (!modulePath) {
    console.error(`Unable to resolve "${commander.preset}" preset`);
    process.exit(1);
  }
} else {
  modulePath = resolveCwd(`./${commander.file}`);

  if (!modulePath) {
    console.error(`Unable to find "${commander.file}" file`);
    process.exit(1);
  }
}

const tasks = require(modulePath);
```

what do you think?

Comment (laggingreflex) [reply lib/index.js]: Yeah it's redundant. Updated

**Comment by laggingreflex** [comment]
It seems this has been merged and version update to 4.1.1 but not been published to npm yet. Just letting you know in case you forgot to actually publish, unless you meant to publish later which is cool. 

Also `babel-cli` has `simple-cli`'s version `4.0.0` which in theory should install version `4.1.1` but it seems yarn still installs `4.0.0` - https://github.com/yarnpkg/yarn/issues/2364. It'd be great if you updated `simple-cli`'s version in `babel-cli` as well. It helps bust cached version.

**Comment by deepsweet** [comment]
oops, published.

>Also babel-cli has simple-cli's version 4.0.0 which in theory should install version 4.1.1 but it seems yarn still installs 4.0.0

that's because `yarn.lock` (only the last one, ie your in current your project) is already respected. try `yarn upgrade` or even install it with `yarn.lock` deleted. so if you install Start and its dependencies in your project for the first time, the latest versions from semver ranges will be installed for sure, but after that it's up to you to bump them every time.

the point of Yarn is that you have to control all the dependencies tree "manually", nothing is updated magically.

**Comment by deepsweet** [comment]
ok, now I see the bug.

**Pull request merged**